elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,142 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
```

Comment:
Should we add a `--pfe-modal--BackgroundColor` variable to allow a hook for designers who want to change the color of the modal?

Reply:
Hmm. I think that makes this overly complicated since we'll get into the issue of text color as well if someone opts for a modal on a dark background. What do you think?

Reply:
The text should be light DOM so it will pick up the theme stylesheet which will applying the broadcast--text variable which can be updated at the same time as the background color.  That does point out that we would need to make the fill color on the close icon pick up the broadcast color though.  Let's do this update as a future enhancement instead of now. 👍

Reply:
Sounds good!

Reply:
Added this issue for tracking that work later: https://github.com/patternfly/patternfly-elements/issues/483